src/js/pages/catalog/PackagesTab.js

```diff
@@ -207,10 +216,10 @@ class PackagesTab extends mixin(StoreMixin) {
     }
 
     let subtitle = (
-      <p className="tall flush-top">
+      <Trans render="p" className="tall flush-top">
         Community packages are unverified and unreviewed content from the
         community.
-      </p>
+      </Trans>
     );
     let title = "Community";
```

Comment:
We could have Community grid translated similarly to the Certified grid above this, if we separated the rendering of the search results to its own function instead of using  `getCommunityPackagesGrid()` to render community packages and search results. I didn't do the refactor now since it would of increased scope of this change.